hedera-node/test-clients/build.gradle.kts

```
@@ -258,6 +275,19 @@ val cleanValidation =
         delete(File(project.file("validation-scenarios"), "ValidationScenarios.jar"))
     }
 
+val cleanRcdiff =
+    tasks.register<Delete>("cleanRcdiff") {
+        group = "build"
+        delete(File(project.file("rcdiff"), "rcdiff.jar"))
```

Works, but cleaner would be:

```suggestion
        delete(layout.projectDirectory.dir("rcdiff"))
```

I think the `"rcdiff.jar"` here has no effect. There is no such file in the root of the proejct.

---

The JAR is in _.gitignore_ but is built here.